src/firebase.ts

```diff
@@ -1,10 +1,10 @@
-import { initializeApp, getApps, getApp } from 'firebase/app';
-import { connectAuthEmulator, getAuth } from 'firebase/auth';
-import { getFirestore, connectFirestoreEmulator } from 'firebase/firestore';
-import { clientConfig } from './auth-config';
-import type { FirebaseApp } from 'firebase/app';
-import type { Auth } from 'firebase/auth';
-import type { Firestore } from 'firebase/firestore';
+import { initializeApp, getApps, getApp } from "firebase/app";
```

Comment:
have you rebased? I think our rules for the code formatter uses single-quotes? This might cause a merge conflict, but not too sure. I think maybe just switching back to single quotes on the safe side might be better

Reply:
Unless the doublequotes is the updated styling